pkg/asset/installconfig/powervs/metadata.go

```diff
@@ -81,7 +76,6 @@ func (m *Metadata) APIKey(ctx context.Context) (string, error) {
 		if err != nil {
 			return "", err
 		}
-
 		m.client = client
 	}
```

Comment:
You can remove this block now, right?

Reply:
This diff is clearly erroneous. I don't have to remove that blank line. Will revert.